.github/workflows/build.yml

```diff
@@ -152,6 +152,25 @@ jobs:
                 -S . -B build
           cmake --build build
 
+  java-binding:
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+      - name: Bootstrap
+        run: |
+          script/bootstrap.sh
+          brew install swig --HEAD
```

Comment:
do we really need to use `HEAD` version of SWIG?

Reply:
We need at least `swig 4.0`, changed to install `swig@4`. Please review. @simonlingoogle